packages/ciscospark/src/plugins/messages.js

```diff
@@ -5,6 +5,7 @@
  */
 
 import {SparkPlugin, Page} from '@ciscospark/spark-core';
+import {isArray} from 'lodash';
```

Comment:
i'm pretty sure lodash is the way to go. last time i looked into it, it seemed like there were a lot of edge cases and broken implementations for Array.isArray